apollo-mockserver/src/jsMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -0,0 +1,38 @@
+package com.apollographql.apollo3.mockserver
+
+actual class MockServer {
+
+  private val responseQueue = mutableListOf<MockResponse>()
+
+  private val server = http.createServer { req, res ->
+    println(req)
+    res.writeHead(200)
+    res.end("Hello, World!")
+  }.listen(PORT)
+
+  actual fun url(): String {
+    //TODO Use `client.address()` but it might return null, before the server is listening. So this will have to be suspend fun
```

Comment:
I think `server.address()` is what we want? The url shouldn't depend on any client being connected. If getting a server address takes some time, I think making it suspend is ok. Most of the tests are run from a coroutine.

Reply:
Oops, my bad. It of course should be `server.address()`. And yes, it returns `null` before it binds to the port.

One nice thing with this being async is that we won't need to specify a port; Node.js will just pick one that's available.

Reply:
Yep, that'll come in handy 👍